app/javascript/packs/application.js

```diff
@@ -16,7 +16,6 @@ import "underscore"
 import {Spinner} from "spin.js"
 import "clipboard"
 import * as Cookies from "js-cookie"
-import "tipsy"
```

Comment:
This should probably import tippy.  Currently I'm getting a `Uncaught ReferenceError: tippy is not defined`.  This might also fix the drag and drop upload error.

Reply:
Yep it does, nice catch!